docs/swift-installation.md

```diff
@@ -180,6 +181,57 @@ defaults write com.apple.dt.Xcode UseSanitizedBuildSystemEnvironment -bool NO
 
 After these steps, after running your tests locally in Xcode, you will get a test report in Scope with the results.
 
+#### SCOPE_DSN
```

Comment:
Doesn't it make more sense to have this just before `Local development integration`?

Reply:
Yes, moved CI env variables before Local development, and left SCOPE_XCODE_DSN after